src/fa/game_process.py

```diff
@@ -97,7 +97,7 @@ def run(self, info, arguments, detach=False, init_file=None):
             logger.info("Running FA via executable: " + executable)
 
             # Launch the game as a stand alone process
-            if not instance.running():
```

Comment:
I'm surprised that this code used to run, I can't find any 'instance' import earlier in the file. There must be some ugly importing done earlier somewhere. We should explicitly import fa so it doesn't mysteriously break in the future.

Reply:
oi .. I might have been sleeping doing that  .. I can't remember putting that 'fa.' there, because 'fa.instance.running' doesn't reference here. 
And no mystery because 'instance' is defined here line 154 ;-)